src/guides/v2.3/payments-integrations/base-integration/integration-intro.md

```diff
@@ -19,9 +19,6 @@ The Magento payment provider gateway allows creating secure and PCI-compliant in
 The topics in this chapter explain how to add an integration with a custom payment service provider (in other words, add a new payment method) and implement the authorize payment action for this [payment method](https://glossary.magento.com/payment-method). For illustration, we use code
 samples from the [Braintree]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Braintree) payment integration.
 
-To simplify the development of a new payment integration, Magento developed the [Payment sample module](https://github.com/magento/magento2-samples/tree/master/sample-module-payment-gateway).
```

Comment:
Revert this deletion and change it to something like

{:.bs-callout-info}
You can also view the [Payment sample module](https://github.com/magento/magento2-samples/tree/master/sample-module-payment-gateway) in the `magento/magento2-samples` 
 repository to understand the underlying principles.  However, be aware that this code is NOT supported.